videomass/vdms_io/make_filelog.py

```diff
@@ -55,6 +55,9 @@ def make_log_template(logname, logdir, mode="a"):
     Returns the absolute/relative pathname of the log
     """
     current_date = time.strftime("%c")  # date/time
+    if logname is None or logname == "":
```

Comment:
This is unrelated to the change but i think it may still be useful to have

Reply:
Hmm, I doubt a log name is an empty string (or other type object) on Videomass. However, the matter is a little more complicated, for instance: On Windows the colons  (":") given by `current_date` is an illegal character for that OS, furthermore even if you would give a name compatible with all operating systems, your solution will not is enough to sanitize a file name.

Anyway, thanks for raising this issue and providing your code as a workaround, you're welcome.

Reply:
If the rest of the PR is valid, please wait until I push a new commit removing this before merging.

The reason why I've added this is because I was getting a crash due to the logname being empty/null, but I guess it must have been an issue with my system.

Reply:
anyway, this is besides the point of this PR :) It may be worth looking a bit further into this potential issue and open another issue to fix it

Reply:
Thank you, but I worked (in my spare time) on it a lot this week and finally fixed this problem too...
This is my PR that should fix it #275

Reply:
Lovely, I'll close my pr and checkout the latest code once you've committed it. Cheers!